### ISSUE TITLE
Chado load problem: duplicated "relationship: " lines in OBO

### ISSUE THREAD

**Issue by kimrutherford** [pombase/fypo#3707]
While testing fixes for #3706 I tried removing the "output_of" Typedef from the "fypo-simple-pombase.obo".  It solved the first problem but triggered a different one.  There are many term definitions that have a pair of relationships like this:

```
 relationship: output_of FYPO:0000032 ! abnormal cytokinesis
 relationship: RO:0002353 FYPO:0000032 ! abnormal cytokinesis
```

RO:0002353 is output_of:
```
 [Typedef]
 id: RO:0002353
 name: output_of
 def: "inverse of has output" []
 is_a: RO:0000056 ! participates in
 is_a: RO:0002328 ! functionally related to
```

This duplicate relationship causes the loader to fall over.

Here's a full example:
```
[Term]
id: FYPO:0000349
name: abnormal Golgi morphology
def: "A physical cellular phenotype observed in the vegetative growth phase of the life cycle in which the size, shape, or structure of the Golgi apparatus is abnormal." [PomBase:mah]
comment: Note that an abnormal Golgi organization process may, but does not necessarily, result in abnormal Golgi morphology.
synonym: "abnormal Golgi morphology during mitotic cell cycle" RELATED [PomBase:mah]
synonym: "abnormal Golgi morphology during vegetative growth" EXACT [PomBase:mah]
synonym: "Golgi morphology: abnormal" RELATED [SGD:phenotype_annotation]
is_a: FYPO:0000353 ! abnormal endomembrane system morphology
relationship: output_of FYPO:0000806 ! abnormal Golgi organization
relationship: RO:0002353 FYPO:0000806 ! abnormal Golgi organization
created_by: midori
creation_date: 2011-07-14T05:04:23Z
```

There are also some duplicates relationships with has_output/RO:0002234:

```
[Term]
id: FYPO:0000174
name: abnormal cell wall biogenesis
def: "A cellular process phenotype in which cell wall biogenesis is abnormal." [PomBase:mah]
synonym: "abnormal fungal-type cell wall biogenesis" EXACT [GO:0009272, PomBase:mah]
is_a: FYPO:0000628 ! abnormal cellular process
relationship: has_output FYPO:0002946 ! abnormal cell wall
relationship: RO:0002234 FYPO:0002946 ! abnormal cell wall
created_by: midori
creation_date: 2011-04-13T03:34:54Z
```

**Comment by mah11** [comment]
ping @matentzn  (also emailed)

**Comment by mah11** [comment]
It looks like @matentzn has fixed the fypo-simple-pombase.obo file, so if it loads nicely we can close this and #3706. (New problems -> new ticket(s).)

**Comment by mah11** [comment]
fixed, as noted in #3706
